### ISSUE TITLE
Narrator + Edge doesn't read the ariaLabel of DetailsList

### ISSUE THREAD

**Issue by YusongLiu** [OfficeDev/office-ui-fabric-react#2490]
### Bug Report

- __Package version(s)__: 4.30.0
- __Browser and OS versions__: Edge latest version

#### Describe the issue:

This is the setting of the DetailsList:

```
        <DetailsList
          **ariaLabel={ ... }**
          checkboxVisibility={ CheckboxVisibility.hidden }
          columns={ ... }
          compact={ true }
          items={ ... }
          layoutMode={ DetailsListLayoutMode.fixedColumns }
          onRenderItemColumn={ ... }
          onRenderRow={ ... }
          selectionMode={ SelectionMode.none }
        />
```

#### Actual behavior:

Narrator + Edge doesn't read ariaLabel when first focusing on the header or list item of the DetailsList.
JAWS + IE also doesn't read.

#### Expected behavior:

Narrator + Edge should read the ariaLabel.
